tests/e2e/test_main.py

```diff
@@ -198,23 +198,26 @@ def test_image_build_custom_preset(
     img_name = f"image:extras-e2e-custom-preset-{rnd}"
     img_uri_str = f"{img_name}:{tag}"
 
-    result = cli_runner(
-        [
-            "neuro",
-            "image-build",
-            "--preset",
-            custom_preset,
-            "-f",
-            str(dockerfile_path),
-            ".",
-            img_uri_str,
-        ]
-    )
-    assert result.returncode == 0, result
-    sleep(10)
+    try:
+        result = cli_runner(
+            [
+                "neuro",
+                "image-build",
+                "--preset",
+                custom_preset,
+                "-f",
+                str(dockerfile_path),
+                ".",
+                img_uri_str,
+            ]
+        )
+        assert result.returncode == 0, result
+        sleep(10)
 
-    result = repeat_until_success(["neuro", "image", "tags", img_name])
-    assert tag in result.stdout
+        result = repeat_until_success(["neuro", "image", "tags", img_name])
+        assert tag in result.stdout
+    finally:
+        cli_runner(["neuro", "image", "rm", img_uri_str])
```

Comment:
in other places, we do `try-catch` and in case of error, just print the error. Maybe even not print as we usually see too many error messages.